packages/execution-environments/src/common/BaseSnapExecutor.ts

```diff
@@ -212,26 +220,23 @@ export class BaseSnapExecutor {
     }
   }
 
+  private registerSnapExports(snapName: string, module: SnapModuleExports) {
+    if (typeof module.exports?.onMessage === 'function') {
```

Comment:
Should we just use the default export instead of requiring a specific name?

Reply:
This allows us to easily extend the API in the future without losing backwards compatibility. Also default exports in Typescript is not the way forward